src/core/public/plugins/plugins_service.ts

```diff
@@ -53,7 +53,7 @@ export interface PluginsServiceStart {
  */
 export class PluginsService implements CoreService<PluginsServiceSetup, PluginsServiceStart> {
   /** Plugin wrappers in topological order. */
-  private readonly plugins = new Map<PluginName, PluginWrapper<unknown, Record<string, unknown>>>();
+  private readonly plugins = new Map<PluginName, PluginWrapper<unknown, unknown>>();
```

Comment:
Had to do that to match the `createPluginSetupContext` type in the call line 117